api/core/repositories/celery_workflow_node_execution_repository.py

```diff
@@ -24,6 +25,17 @@
     save_workflow_node_execution_task,
 )
 
+# Safe field getter mapping for WorkflowNodeExecution to avoid getattr reflection
+WORKFLOW_NODE_EXECUTION_FIELD_GETTERS = {
+    "id": lambda x: x.id,
```

Comment:
imo `dict[str, Callable[[ the type name ], Any]]` is better.

btw now it is called Row polymorphism

ref https://www.cl.cam.ac.uk/teaching/1415/L28/rows.pdf

Reply:
> imo `dict[str, Callable[[ the type name ], Any]]` is better.
> 
> btw now it is called Row polymorphism
> 
> ref https://www.cl.cam.ac.uk/teaching/1415/L28/rows.pdf

Done.